CHANGELOG

```diff
@@ -27,6 +28,10 @@ Qtile 0.x.x, released xxxx-xx-xx:
         - added new argument for CheckUpdates widget: `custom_command_modify` which allows user to modify the
           the line count of the output of `custom_command` with a lambda function (i.e. `lambda x: x-3`).
           Argument defaults to `lambda x: x` and is overridden by `distro` argument's internal lambda.
+        - A Wayland backend has been added which can be used with `qtile start -b wayland`. It is undergoing heavy development
+          and bugs are to be expected, but if you find any bugs or conspicuously missing features that are not listed on our
+          to do list at https://github.com/qtile/qtile/discussions/2409 please do let us know. The backend requires pywlroots
+          and its dependencies python-xkbcommon, pywayland and wlroots 0.13.0.
```

Comment:
I would mention that any wayland-specific configuration is not intended to be stable at this time.